app/src/components/features/rules/RuleBuilder/Header/ActionButtons/CreateRuleButton/actions/index.js

```diff
@@ -403,19 +404,21 @@ export const ruleModifiedAnalytics = (user) => {
 };
 
 export const fixSourceRegexFormat = (dispatch, rule) => {
```

Comment:
Remove this function from this file, as its already imported from `utils/rules/misc.js`